### DISCUSSION TITLE
how to install tinytex offline or config tinytex already exists?

### DISCUSSION THREAD

**Discussion by plmcoder** [quarto-dev/quarto-cli#6444] [Q&A]
### Description

Hi,

I'd like to use Quarto with conda in an internal compute environment without network.
but `quarto install tinytex` command will not work without internet.

so I download latest [TinyTeX-1-v2023.08.zip](https://github.com/rstudio/tinytex-releases/releases/download/v2023.08/TinyTeX-1-v2023.08.zip) from https://github.com/rstudio/tinytex-releases
Is it possible to config external TinyTeX to quarto, what is the command?

```
OS: win 10 64bit
Conda: anaconda 2023.7-2
```
BTW, I found I could not get full help from `quarto help` command and could not find any cli document/instructions.
could anybody help me?

**Comment by dragonstyle** [comment]
In general, Quarto attempts to use TinyTex in its default installed path, or will use a TeXLive installation that is on the system path (e.g. `tlmgr`). Quarto will use its own installation of TinyTex in preference to the system installation of TeX, but other than specifically looking for that, it should just inherit whatever is available. Are you not seeing this behavior?

**Reply by plmcoder** [reply]
dragonstyle,  thanks a lot for your reply.

I found a way to make it work.  do as follows:
1. download TinyTeX separately.
2. extract and place it in `C:\Users\alex\AppData\Roaming\TinyTeX`.
then it could be recognized by quarto. 
```cmd
(base) C:\Users\alex\Desktop\test>quarto list tools
[>] Inspecting tools

Tool         Status                    Installed     Latest
chromium     Not installed             ---           869685
tinytex      External Installation     ---           v2023.08
```
But this doesn't seem very friendly, I wonder if there is a better way, such as setting to `D:\apps\TinyTex`, or directly packaging it in the quarto installation directory?

**Reply by dragonstyle** [reply]
We inspect the default installation paths for TinyTex specifically (assuming the user used some other mechanism to install it), or we will use TexLive found on the path (meaning it could be installed anywhere). If you'd like you can place it in `apps\TinyTex` and add the proper bin directory to your path and we should support that as well.

We don't package it directly in the installer as it will more than double the size of the installer and not all users need the capability to generate PDFs to be installed (either they are not producing PDFs with Quarto or they already have an installation of TeX that they'd rather use).
